libs/blocks/section-metadata/section-metadata.js

```diff
@@ -1,17 +1,28 @@
 function handleBackground(div, section) {
-  const pic = div.querySelector('picture');
+  const pic = div.background.content.querySelector('picture');
   if (pic) {
     section.classList.add('has-background');
     pic.classList.add('section-background');
+    if (div.focalpoint) {
```

Comment:
I think this should be a little more hardened. If an author enters "focal point" this breaks.

Reply:
This approach has been removed. Directly the value for focal point would be given under the image.